http/rest-client-reactive/src/main/java/io/quarkus/ts/http/restclient/reactive/resources/TLSResource.java

```diff
@@ -0,0 +1,29 @@
+package io.quarkus.ts.http.restclient.reactive.resources;
+
+import jakarta.enterprise.event.Event;
+import jakarta.inject.Inject;
+import jakarta.ws.rs.POST;
+import jakarta.ws.rs.Path;
+import jakarta.ws.rs.PathParam;
+
+import io.quarkus.tls.CertificateUpdatedEvent;
+import io.quarkus.tls.TlsConfiguration;
+import io.quarkus.tls.TlsConfigurationRegistry;
+
+@Path("/tls")
+public class TLSResource {
+
+    @Inject
+    Event<CertificateUpdatedEvent> event;
+    @Inject
+    TlsConfigurationRegistry registry;
+
+    @POST
+    @Path("/reload/{name}")
+    public void reload(@PathParam("name") String name) {
+        TlsConfiguration config = registry.get(name).orElseThrow();
+        if (config.reload()) {
```

Comment:
If `reload` returns `false`, it means the reload you wanted to happen didn't. if you didn't put it into the `if` I'd not comment because I expect you can recognize whether reload happened based on the test results... but as you added  `if`, I think this invocation should result in illegal state if `reload` returns `false`